ngprc

```diff
@@ -16,6 +16,18 @@ editor = "vim -c 'set hls' -c 'silent /%1$s' -c %2$d %3$s"
 //editor = "emacs +%2$d %3$s &"
 //editor = "subl %3$s:%2$d 1>/dev/null 2>&1"
 
+// default parser: nat, ag or git
```

Comment:
Maybe we can add `native` like below to make sure it's clear 
`// default parser: nat (native), ag or git`